### PR TITLE
Dev dfm/issue7

### LINKED ISSUE #7: Component - Navigation Bar
### Screenshot
image

### User Story
As a user
I want to see a navigation bar
so I can easily move within the app to the different functionalities. 

### To Do
- [ ] Radiobuttons and link
- [ ] `Components` for every icon (Home, Profile, Add, Search, Settings)
- [ ] `Component` Navbar with all icons
- [ ] Drop animation for selected icon

### Acceptance Criteria

- Given I am on the Group List Page (???), then I should see a navigation bar at the button containing the icons described above.
- Given I click on one specific icon, then I see the icon is selected as per the attached design.

### PR DESCRIPTION
**Pull request by Dev-dfm**
- Add component for NavBar Icons
- Add NavBar component
- Implement NavBar Animation

Try it out:
https://fit-2-gether-dev-dfm-is-5wsslg.herokuapp.com/storybook/?path=/story/component-navbar--nav-bar-container

fixes #7

### CODE REVIEW

File: src/app/components/NavBar/NavBar.module.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.navBar {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: var(--secondary-text-color);
+  padding: 0.5em 0;
+  position: relative;
+  width: 500px;
```
Comment (phlgr) [inline src/app/components/NavBar/NavBar.module.css:8]: The fixed width is needed for the drop placement?

File: src/app/components/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+} = {
+  '/main': -187,
+  '/profile': -93,
+  '/add': 0,
+  '/search': 93,
+  '/': 187,
+};
+export default function NavBar(): JSX.Element {
+  // eslint-disable-next-line @typescript-eslint/no-inferrable-types
+  const location = useLocation();
```
Comment (phlgr) [inline src/app/components/NavBar/NavBar.tsx:23]: Can you remove this eslint-disable rule?

File: src/app/components/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import { useLocation } from 'react-router-dom';
+
+const translateDrop: {
+  [path: string]: number;
+} = {
+  '/main': -187,
+  '/profile': -93,
+  '/add': 0,
+  '/search': 93,
+  '/': 187,
```
Comment (phlgr) [inline src/app/components/NavBar/NavBar.tsx:19]: Could this be translated to percentages?

**Review by dhnazarenko — COMMENTED** [review]
Really cool! 🔥

File: src/app/components/NavBar/NavBar.module.css (hunk 1 above)
Comment (Dev-dfm) [reply src/app/components/NavBar/NavBar.module.css:8]: Yes, but only for the moment until I find a solution to adapt it to the viewport.

**Review by lajanzen — APPROVED** [review]
Wow!! Really REALLY nice!!! 😱🚀

**Pull request merged**